### PR TITLE
fix duplicate account names 

### LINKED ISSUE #505: Accounts can have the same name
I'm unsure if this is a feature or a bug. 

However having multiple accounts with the same name seems like a terrible idea. Particularly as they can't be renamed.

Suggested solution:

1) Don't allow while renaming accounts is not possible
2) Allow with a warning if renaming accounts is enabled

### PR DESCRIPTION
**Pull request by jakeols**
A simple fix #505. Disables the next button when an account with that name already exists.

### CODE REVIEW

**Comment by jakeols** [comment]
[clabot:check]

**Comment by jakeols** [comment]
[clabot:check]

File: packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js [hunk 1]
```diff
@@ -118,6 +120,14 @@ class AccountName extends Component {
       location: { pathname }
     } = this.props;
     const currentStep = pathname.slice(-1);
+    const { accountsInfo } = this.props;
+
+    let accountNameExists = false;
```
Comment (amaurymartiny) [inline packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js]: You can use `.find()` here

```typescript
const accountNameExists = !!Object.values(accountsInfo).find((info) => info.name === name)
```

**Comment by jakeols** [comment]
Makes sense - I added the message (not sure if there's a specific convention for styling).

File: packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js [hunk 2]
```diff
@@ -115,9 +117,14 @@ class AccountName extends Component {
...
       error,
       history,
-      location: { pathname }
+      location: { pathname },
+      accountsInfo
     } = this.props;
+
     const currentStep = pathname.slice(-1);
+    const accountNameExists = !!Object.values(accountsInfo).find(
+      info => info.name === name
```
Comment (Tbaut) [inline packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js]: I would even go further and perform a case unsensitive matching
```suggestion
      info => info.name.toLowerCase() === name.toLowerCase()
```

**Comment by amaurymartiny** [comment]
Thanks for the PR!

**Pull request merged**